src/Raven.Studio/typescript/commands/database/documents/counters/deleteCounterCommand.ts

```diff
@@ -0,0 +1,36 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class deleteCounterCommand extends commandBase {
+
+    constructor(private counterName: string, private documentId: string, private db: database) {
+        super();
+    }
+
+    execute(): JQueryPromise<Raven.Client.Documents.Operations.Counters.CountersDetail> {
+
+        const payload = {
```

Comment:
can we introduce some server side type here? CounterBatch ??

Reply:
todo: Add as a type.....

Reply:
done